.github/workflows/lint.yaml

```
@@ -256,3 +256,25 @@ jobs:
         uses: codespell-project/codespell-problem-matcher@v1
       - name: codespell
         run: ./scripts/spelling.sh codespell
+  weblint:
+    name: weblint
+    runs-on: ubuntu-latest
+    container: debian:stable
+    steps:
+      - name: Update package database
+        run: apt-get update -y
+      - name: Install Git
+        run: apt-get -y install git
+      - uses: actions/checkout@v3
+      - name: Setup Node.js v18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18 # LTS
+          cache: 'npm'
+          cache-dependency-path: './javascript/new-src/package-lock.json'
+      - name: Install Node.js lint tools and dependencies
```

Technically, this step installs "npm dependencies", not "Node.js" dependencies.

```suggestion
      - name: Install npm dependencies
```